src/hooks/useCart/useCart.tsx

```diff
@@ -249,7 +250,7 @@ export const useCart = (
   const updateCart: CartHook["updateCart"] = useCallback(
     (category, quantity, identifierInputs) => {
       if (quantity < 0) {
-        setError(new Error("Invalid quantity"));
+        setError(new Error(ERROR_MESSAGE.INVALID_QUANTITY));
```

Comment:
This works well for now. Am wondering if it's useful to throw a special `CartError` instead? Then we can scope the error messages to be under `CartError` i.e. `new CartError(INVALID_QUANTITY)`. Perhaps the alert props can be inside the error too.

Reply:
good idea for this! will add this as a task for the bigger PR, thanks!